tests/acceptance/features/bootstrap/WebUISharingContext.php

```diff
@@ -896,6 +896,22 @@ public function theUsersOwnNameShouldNotBeListedInTheAutocompleteList() {
 		);
 	}
 
+	/**
+	 * @Then the user :username should not be listed in the autocomplete list on the webUI
+	 *
+	 * @param string $username
+	 *
+	 * @return void
+	 *
+	 * @throws Exception
+	 */
+	public function theUserShouldNotBeListedInTheAutocompleteListOnTheWebui($username) {
+		$names = $this->sharingDialog->getAutocompleteItemsList();
+		if (\in_array($username, $names)) {
+			throw new Exception("$username should not be present in the autocomplete items list");
```

Comment:
Error message should tell what the current state is, not what needs to be. 
Below is just a basic suggestion, needs to be improved.
```suggestion
			throw new Exception("$username was found in the autocomplete items list, but was not expected.");
```